scripts/install_deepchem_conda.ps1

```diff
@@ -8,18 +8,7 @@ else
     $python_version=3.6
 }
 
-if($args.Length -eq 1)
```

Comment:
Currently, we support only `deepchem` virutal  environment because `requirements.yml` support only deepchem env.

See: https://github.com/deepchem/deepchem/blob/master/requirements.yml